docs/docs/get-started/README.md

```diff
@@ -1,3 +1,106 @@
+VAST is an embeddable security telemetry engine for structured event data.
+Tailor-made for security operations, VAST is the foundation for many data-driven
+detection and response uses cases, such as operationalizing threat intelligence,
+threat hunting, event contextualization, and advanced detection engineering.
+
+:::tip Wanna give it a try?
+The [quick start guide](/docs/get-started/quick-start-guide) shows you how to
+get up and running and gives a quick tour of the key use cases.
+:::
+
+The documentation has the following key sections:
+
+1. [Setup VAST](/docs/setup-vast) describes how you can download, install, and
+   configure VAST in a variety of environments.
+   👉 *Start here if you want deploy VAST.*
+2. [Use VAST](/docs/use-vast) explains how to work with VAST, e.g., ingesting
+   data, running queries, matching threat intelligence, or integrating it with
+   other security tools.
+   👉 *Go here if you have a running VAST, and want to explore what you can do
+   with it.*
+3. [Understand VAST](/docs/understand-vast) describes the system design goals
+   and architecture, e.g., the actor model as concurrency and distribution
+   layer, separation of read/write path, and core components like the catalog
+   that provides light-weight indexing and manages schema meta data.
+   👉 *Read here if you want to know why VAST is built the way it is.*
+4. [Develop VAST](/docs/develop-vast) provides developer-oriented resources to
+   customize VAST, e.g., write own plugins or enhance the source code.
+   👉 *Look here if you are ready to get your hands dirty and write code.*
+
+If you're unsure whether VAST is the right tool for your use case, keep reading
+down below.
+
+## Why VAST?
+
+VAST fills a gap when you need a highly embeddable database for security
+data that powers detection and response use cases. The following graphic
+illustrates the placement of VAST in the spectrum of *Observability* ⇔
+*Security* and *Open Platform* ⇔ *Data Silo*.
+
+![VAST Spectra](/img/ecosystem.png)
+
+We compare VAST to SIEM and data warehouses next, but skip a comparison with
+metrics services that we deem out of scope.
+
+### VAST vs. SIEM
+
+VAST *complements* a [SIEM][siem] nicely with the following use cases:
+
+- **Offloading**: route the high-volume telemetry to VAST that would otherwise
+  overload your SIEM or be cost-prohibitive or to ingest. By keeping the bulk of
+  the data in VAST, you remove bottlenecks and can selectively forward the
+  activity that matters to your SIEM.
+
+- **Compliance**: VAST has fine-grained retention span configuration to meet
+  GDPR and other regulatory requirements. When storage capacity needs careful
+  management, VAST's *compaction* feature allows for weighted ageing of your
+  data, so that you can specify relative importance of event types. Powerful
+  *transforms* allow you to anonymize, pseudonomyize, or encrypt specific
+  fields—either to sanitize PII data on import, or ad-hoc on export when data
```

Comment:
If we have links to siem and arrow, we might want to have one for the other acronyms as well, depending on the target audience PII may not be immediately obvoius.